torch/csrc/jit/script/sugared_value.cpp

```diff
@@ -339,6 +339,15 @@ RangeValue::RangeValue(
     const SourceRange& loc,
     Function& m,
     std::vector<Value*> inputs) {
+  for (size_t i = 0; i < inputs.size(); ++i) {
+    auto typ = inputs[i]->type();
+    if (!typ->cast<IntType>()) {
+      throw ErrorReport(loc)
+          << "all inputs of range must be ints, found " << typ->python_str()
+          << " in argument " << c10::guts::to_string(i);
```

Comment:
what is `c10::guts` package and why we need this?

Reply:
this is the recommended way to turn an integer into a string so that it works on all platforms

Reply:
it's our internal way of converting values to strings that works on all platforms. 
`ctrl-f c10::guts::to_string` : 19 results in our library

Reply:
to_string is not needed, just i.